third-party/qthread/Makefile

```diff
@@ -113,9 +113,9 @@ CHPL_QTHREAD_CFG_OPTIONS += --enable-static --disable-shared
 
 # determine which scheduler to use. Use nemesis for flat and distrib for
 # non-flat.  Override with a user provided option if they requested one
-SCHEDULER = nemesis
-ifneq ($(CHPL_MAKE_LOCALE_MODEL),flat)
 SCHEDULER = distrib
+ifneq ($(CHPL_MAKE_LOCALE_MODEL),numa)
```

Comment:
I've updated the comment to read:

```
# determine which scheduler to use. Use nemesis for numa and distrib for any
# other model (e.g. flat and gpu).  Override with a user provided option if
# they requested one
```

Reply:
I think that comment is backwards. We want distrib for numa, but nemesis for everything else. Maybe the code would be more obvious about which is the default if we switched it to something like:

```make
SCHEDULER = nemesis
ifeq ($(CHPL_MAKE_LOCALE_MODEL),numa)
  SCHEDULER = distrib
endif
```

Reply:
🤦 whoops, thanks for catching that.

More generally, it would be better for that comment to explain why that particular scheduler is used for the NUMA locale model rather than repeating what the code says (incorrectly in my previous commit).

If you happen to know let me know and I can update. Otherwise I've changed it to:

```
# determine which scheduler to use based on locale model.  Override with a user
# provided option if they requested one
SCHEDULER = nemesis
ifeq ($(CHPL_MAKE_LOCALE_MODEL),numa)
  SCHEDULER = distrib
endif
```

Thanks!

Reply:
Ok thanks. I've copied that info in this commit:

https://github.com/chapel-lang/chapel/pull/19857/commits/9e6bf783f7071dc45568eb2cd43f06a21b101e0d